src/index.js

```diff
@@ -1,8 +1,9 @@
 import PaintEditor from './containers/paint-editor.jsx';
+import CopyPasteHOC from './containers/copy-paste-hoc.jsx';
 import SelectionHOC from './containers/selection-hoc.jsx';
 import ScratchPaintReducer from './reducers/scratch-paint-reducer';
 
-const Wrapped = SelectionHOC(PaintEditor);
+const Wrapped = SelectionHOC(CopyPasteHOC(PaintEditor));
```

Comment:
When you move `CopyPasteHOC` into `PaintEditor`, please also move `SelectionHOC` for consistency.

Reply:
🍨